include/uv.h

```diff
@@ -706,10 +706,18 @@ typedef enum {
   UV_TTY_MODE_IO
 } uv_tty_mode_t;
 
+typedef enum {
+  UV_TTY_SUPPORTED,
+  UV_TTY_UNCHECKED,
```

Comment:
Undocumented value. If you want a tri-state for internal use, you should try to come up with something else.